VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioComponentDetails.cpp

```diff
@@ -96,6 +98,23 @@ bool IsVitruvioComponentSelected(const TArray<TWeakObjectPtr<UObject>>& ObjectsB
 	return false;
 }
 
+void GetAllVitruvioComponents(const TArray<TWeakObjectPtr<UObject>>& ObjectsBeingCustomized, 
```

Comment:
Consider returing the ObjectsBeingCustomized by value from the function instead of passing by reference (would also make the callsite a bit easier to understand).